recipe/meta.yaml

```
@@ -48,10 +48,12 @@ requirements:
     #  Please don't distribute binary packages of libarchive linked
     #  against liblzo."
     # - lzo
-    - openssl  # [not osx]
+    - openssl {{ openssl }} # [not osx]
     - libxml2 2.10
```

```suggestion
    - libxml2 {{ libxml2 }}
```
As we already have 2.10 in cbc.yaml https://github.com/AnacondaRecipes/aggregate/blob/a6279499266b2a4b9f12f33617275882b078a015/conda_build_config.yaml#L131-L132